.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Install
         run: yarn
       - name: Build
```

Comment:
I'd prefer to have two separate steps. It'll make it simpler to see where we had an error. See my comments below.

```suggestion
      - name: Lint
        run: npm run lint
      - name: Build
```

Reply:
You're right, having two separate steps would be better.